monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/TypeFilter.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from 'react';
```

Comment:
I would prefer a general dropdown component and feed it with the options

Reply:
Because it's a plugin type filter (it's in `/plugins-marketplace`). Other filters might require different logic. I think it's not worth it to try and generalize it beyond handling installed AND updated plugins for now